app/models/user.rb

```diff
@@ -78,12 +78,12 @@ def four_different_characters
 
   has_many :team_users
   has_many :teams, through: :team_users, source: :team
-  has_many :non_single_teams, -> { where single: false }, through: :team_users, source: :team
+  has_many :teams_created_by_event, -> { where created_by_event: false }, through: :team_users, source: :team
```

Comment:
@lheilmann can you explain this line to me?
at first sight, it seems paradoxical to me that this lines reads `where created_by_event: false`. i might be missing something here, but that might also indicate that readibility could be improved?

Reply:
You're right @tomkellog – thanks for noticing! I didn't notice before but the variable is actually named wrongly. Only the “real” teams should be displayed on the Teams index page (see #488). The current variable name implies that the opposite—the ”fake“ teams created by events—is displayed on the Teams index page (see `/views/teams/index.html.erb`).

@lheilmann Please change the variable name to something like `teams_not_created_by_event`. Thanks!

Reply:
You're both right - thanks for noticing! Already changed to `created_by_user` and refactored.